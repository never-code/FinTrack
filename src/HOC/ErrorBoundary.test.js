import React from 'react';
import { render } from '@testing-library/react';
import Error from './ErrorBoubdray';

describe('SliderContainer ', () => {
  it('should render successfully', () => {
    const { container } = render(
      <Error>
        <span>Test</span>
      </Error>
    );
    expect(container).toMatchSnapshot();
  });
});

