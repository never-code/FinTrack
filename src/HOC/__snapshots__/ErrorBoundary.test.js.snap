// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliderContainer  should render successfully 1`] = `
<div>
  <span>
    Test
  </span>
</div>
`;
